### DISCUSSION TITLE
Permission denied when saving changes in Python site-packages

### DISCUSSION THREAD

**Discussion by yeetus1992** [openai/whisper#1008] [Q&A]
Permission denied when saving changes in Python site-packages

Hi!

when i go into my python site packages via finder and try to change something in a file and saving that:
`Do you want to save /Library/Python/3.9/site-packages/whisper/decoding.py before closing?`
it gives me:
```
[Errno 13] Permission denied: '/Library/Python/3.9/site-packages/whisper/decoding.py/'
How do I fix this?
```
Would be beyond grateful for help!

Sincerely:)

**Comment by jongwook** [comment]
Duplicate of #1011
